src/components/idleMembers/card/index.tsx

```diff
@@ -1,17 +1,22 @@
 import { FC } from 'react';
 import classNames from '@/components/idleMembers/card/card.module.scss';
 
+const dummyProfile = 'dummyProfile.png';
+const url = process.env.NEXT_PUBLIC_GITHUB_IMAGE_URL;
+
 type Props = {
   idleMemberUserName: string
 }
 
 const Card: FC<Props> = ({ idleMemberUserName }) => {
-  const imageGenerator = (name: string) => `${process.env.NEXT_PUBLIC_GITHUB_IMAGE_URL}/${name}/img.png`;
-
+  const assigneeProfilePic = (name: string) => `${url}/${name}/img.png`;
   const getMemberDetails = (name: string) => {
     const newWindow = window.open(`https://members.realdevsquad.com/${name}`, '_blank', ' noopener ,norefferrer');
     if (newWindow) newWindow.opener = null;
   };
+  const assigneeImageOnError = (e: any) => {
```

Comment:
why `any`? Please avoid using `any`, it totally defeats the purpose of using typescript.

Here `e` can be - `SyntheticEvent<HTMLDivElement>` (Just re-check once).
Also, replace `e` with `events`.